cis194/week4/daniel-deng/src/Exercise01.hs

```diff
@@ -0,0 +1,22 @@
+module CIS194.Homework04.Exercise01 (fun1, fun2) where
+
+import Data.Bool
+
+fun1 ::  [Integer] -> Integer
+fun1 = product . fmap (subtract 2) . filter even
+
+fun2 :: Integer -> Integer
+fun2 = sum
+     . filter even
+     . takeWhile (>1)
+     . iterate (bool <$> handleOdd <*> handleEven <*> even)
```

Comment:
@shineli1984 I think I'll still use applicative style here just to develop my muscle memory 😂

Reply:
That's fine. Your call.